README.md

```diff
@@ -12,9 +12,16 @@ holds a list of 128-bit GATT UUIDs that
 websites using the Web Bluetooth API are forbidden from accessing.
 This includes all of Services, Characteristics, and Descriptors.
 
-This file contains comments in lines starting with `#`.
-Non-comment lines contain a [valid UUID](https://webbluetoothcg.github.io/web-bluetooth/#valid-uuid)
+[manufacturer_data_blocklist.txt](/manufacturer_data_blocklist.txt)
+holds a list of manufacturer data that websites using the Web Bluetooth API are forbidden from accessing.
+Each entry of the list is consist of company identifier and data prefix that should be blocked.
+
+These files contain comments in lines starting with `#`.
+Non-comment lines contain one of these:
+  * A [valid UUID](https://webbluetoothcg.github.io/web-bluetooth/#valid-uuid)
 followed optionally by a space and one of the tokens "`exclude-reads`" or "`exclude-writes`".
+  *  The word `manufacturer` followed by a space and a group of hexadecimal digits (at most four hex digits)
+  that represents a company identifier. A space after that, is `advdata-` followed by a data prefix with mask.
```

Comment:
Can we cite a registry of company identifiers?

Reply:
Done.